misc/python/materialize/cloudtest/k8s/redpanda.py

```diff
@@ -34,7 +34,7 @@ def __init__(
         super().__init__(namespace)
         container = V1Container(
             name="redpanda",
-            image="vectorized/redpanda:v21.11.13",
+            image="vectorized/redpanda:v23.1.9",
```

Comment:
Perfect, this version is in line with https://github.com/MaterializeInc/materialize/pull/23567/files.